File: 프로그래머스/lv0/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        int answer = 0;
        
        int ant1 = 5;
        int ant2 = 3;
        int ant3 = 1;
        
        while(hp != 0) {
            if(hp >= 5) {
                answer += hp / 5;
                hp %= 5;
            } else if(hp >= 3) {
                answer += hp / 3;
                hp %= 3;
            } else {
                answer += hp / 1;
                hp %= 1;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.py
def solution(hp):
    answer = 0

    while (hp > 0):
        if hp >= 5:
            answer += hp // 5
            hp -= 5 * (hp // 5)
        if hp >= 3:
            hp -= 3
            answer += 1
        if hp >= 1:
            hp -= 1
            answer += 1

    return answer
